Automattic/jetpack
# Contact Form Block: Support flex field widths

Expands on #15362

Update the contact form block to allow field widths to be set. This allows more complex layouts to be achieved:

<img width="1147" alt="Screen Shot 2020-05-04 at 3 37 36 PM" src="https://user-images.githubusercontent.com/1464705/81020242-39044b80-8e1d-11ea-9e4b-f0c450aaafff.png">

**TODO**
- [x] Support flexbox rendering on the front end

#### Testing instructions:






* Add a Contact Form block
* Add a number of different types of fields
* Change the width setting of these fields and confirm they adjust as expected.
* Publish/update the post and confirm these settings and the layout are retained correctly.
* Confirm the same flex layout is rendered on the front end of the site.

#### Proposed changelog entry for your changes:

* Added greater support for flexible layouts using the Contact Form block.
